go.mod

```diff
@@ -80,10 +80,10 @@ require (
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/golangplus/testing v1.0.0 // indirect
 	github.com/google/cadvisor v0.39.0
-	github.com/google/tcpproxy v0.0.0-20180808230851-dfa16c61dad2
 	github.com/google/uuid v1.2.0
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
+	github.com/inetaf/tcpproxy v0.0.0-20200125044825-b6bb9b5b8252
```

Comment:
Per the docs, it appears that this it intended to be imported as `inet.af/tcpproxy`

Reply:
You are right, I resubmitted the code (github.com/inetaf/tcpproxy --> inet.af/tcpproxy), please check it, thank you!

Reply:
It looks like this got changed back when you removed the generated bindata. Can you make sure we're getting the right imports?

Reply:
The previous commit is not the latest, committed again.